h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonImportExportTest.java

```diff
@@ -568,5 +568,19 @@ public void testReadComplexFile() throws Exception {
         stat.execute("DROP TABLE IF EXISTS TABLE_POINTS_READ");
         stat.close();
     }
+
+    @Test
+    public void testReadProperties() throws Exception {
+        Statement stat = connection.createStatement();
+        stat.execute("DROP TABLE IF EXISTS TABLE_PROPERTIES_READ;");
+        stat.execute("CALL GeoJsonRead('D:\\users\\pham\\Desktop\\data.geojson','TABLE_PROPERTIES_READ');");
```

Comment:
The resource folder should be place in the maven `resource` folder. Then the file can be accessed by using `ClassName.class.getResource(String relativeResourcePath)` which return the URL of the resource. Be careful, the base of the relative resource path is the path of the class used.